### PR TITLE
Server setup

### PR DESCRIPTION
**Pull request by Shakib448**

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (Momenuzzaman) [inline server/package.json:7]: habi jabi

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  },
+  "keywords": [],
```
Comment (Momenuzzaman) [inline server/package.json:9]: buge

**Review by Momenuzzaman — COMMENTED** [review]
thik koro

File: server/package.json (hunk 1 above)
Comment (Shakib448) [reply server/package.json:7]: Bepar t thik ase

File: server/package.json (hunk 2 above)
Comment (Shakib448) [reply server/package.json:9]: okh solved

**Review by Momenuzzaman — APPROVED** [review]

**Pull request merged**